app/javascript/readingList/readingList.jsx

```diff
@@ -88,14 +89,9 @@ export class ReadingList extends Component {
     event.preventDefault();
 
     const { statusView, items, totalCount } = this.state;
-    window.fetch(`/reading_list_items/${item.id}`, {
+    request(`/reading_list_items/${item.id}`, {
       method: 'PUT',
-      headers: {
-        'X-CSRF-Token': window.csrfToken,
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({ current_status: statusView }),
-      credentials: 'same-origin',
+      body: { current_status: statusView },
```

Comment:
This doesn't need to be stringified as out `request` utility function handles this internally.

https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L35-L37